pytest.ini

```diff
@@ -34,7 +34,7 @@ python_functions = test*
 env =
     D:ARKOUDA_SERVER_HOST=localhost
     D:ARKOUDA_SERVER_PORT=5555
-    D:ARKOUDA_FULL_STACK_TEST=True
+    D:ARKOUDA_RUNNING_MODE=GLOBAL_SERVER
```

Comment:
At least for CI and nightly testing, `CLASS_SERVER` is probably our preferred mode. The startup overhead is negligible for us, and the benefit to isolating errors down to a single class when something kills the server is beneficial.

My preference would be to leave `CLASS_SERVER` the default for that reason, but if others prefer `GLOBAL_SERVER`, I can update CI and nightly to override.

Reply:
@ronawho I apologize, I am not following what you're writing here. The test cases are executed sequentially, so not sure how GLOBAL_SERVER vs CLASS_SERVER impacts tracing down a test case failure.

Reply:
If the first test class does something that kills the server we lose the remaining testing. For example https://github.com/Bears-R-Us/arkouda/issues/1093 was causing failures in nightly testing for 3 or 4 different classes, but in the `GLOBAL_SERVER` world we only would have seen the first real failure and all other tests would have failed because the server wasn't running.

Reply:
@ronawho ah, I see, yes, you are correct

Reply:
I have two questions:

1. Do we still have a mode to connect to an already running server?
2. Are we going to make the default `CLASS_SERVER`?

Reply:
@mhmerrill made the changes, pulled in upstream master, built/tested pushed to PR, CI successful, ready for review